app/Commands/EnableCommand.php

```diff
@@ -25,10 +24,20 @@ public function handle(): void
             return;
         }
 
-        $option = $this->menu('Services to enable', $this->enableableServices())
-            ->addLineBreak('', 1)
-            ->setPadding(2, 5)
-            ->open();
+        $option = $this->menu('Services to enable:')->setTitleSeparator('=');
+
+        foreach ($this->enableableServicesByCategory() as $category => $services) {
+            $menuItems = collect($services)->mapWithKeys(function ($service) {
+                return [$service['shortName'] => $service['name']];
+            })->toArray();
+
+            $option->addStaticItem("{$category}:")
+                ->addStaticItem('---------')
```

Comment:
Nice! Quick visual refinement suggestion:

Custom separator length for each category to Tighten (pun intended) everything up!

```php
$separatorLength = strlen($category) + 1;
$separator = str_repeat('-', $separatorLength);

$option->addStaticItem("{$category}:")
    ->addStaticItem($separator)
    […]
```

Reply:
Thanks for suggestion.